### ISSUE TITLE
create `manage.py shell` command for dev agility

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#3627]
## Description

When a dev is working on the SecureDrop web applications, it would be useful to be able to minimize the number of steps from `bin/dev-shell` (to hop in the dev container) to being at a prompt where one can interactively run ORM queries using the Flask app context (it doesn't matter which one since they share `models.py`). Flask 1.0 [supports `flask shell`](http://flask.pocoo.org/docs/1.0/cli/#open-a-shell), so we should probably just upgrade to 1.0 and use that. 

tl;dr I want to be able to run e.g. `bin/dev-shell flask shell` and be in an interactive shell with the application context loaded

## User Research Evidence

This is an extremely common workflow for anyone with other Python webapp experience: e.g. Django devs (via `python manage.py shell`) and Flask 1.0+ (via `flask shell`)

## User Stories

As a developer, I want to be able to quickly get a shell with the Flask app context so that my workflow is as streamlined as possible.

**Comment by ageis** [comment]
Have done this before in an app I've been developing. Just to flesh it out a bit further... For your purposes, I could imagine adding something akin to the following method to create_app() in __init__.py within the journalist and source apps. As an example...
```
    @app.shell_context_processor
    def make_shell_context():
# stuff you want to do on the shell, like explore the DB e.g.  
        return {'Source': Source, 'Journalist': Journalist, 'Submission': Submission, 'Reply': Reply}
```
This of course depends on those models having an appropriate __repr__, which in order to fulfill this ticket you may need to change in order to expose more information (columns), plus the name of the `FLASK_APP` perhaps stored in `.flaskenv`. One of my questions is, are there any custom  commands or procedures you'd like to be part of the Flask shell? That would possibly an opportunity to use [click](https://pypi.org/project/click/) (already part of develop-requirements) to make the CLI interface nicer. Or is popping one into an interactive Flask shell with the app context loaded enough (which in theory makes this task fairly straightforward)?

Secondly, I guess I'm confused about whether you want to do this through manage.py or flask shell. The workflow where one exports either FLASK_APP=(source|journalist) and then runs `flask shell` is intuitive for me to grasp—after all there are two apps here, although one shared database, except manage.py has this whole method for importing the Flask app context (asking since unsure—that allows one to use app_context() as a decorator, right? so the above would become @app_context.shell_context_processor instead, and be defined right below that) but the other thing is that manage.py _only_ imports the journalist app, and don't we want devs to be able to obtain a shell with the source app as well? I guess if any of these considerations can be addressed for me then I might be clear enough on this in order to submit an implementation .
